<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>in2pre</class>
 <widget class="QDialog" name="in2pre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>348</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>348</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>574</width>
    <height>348</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Show Step</string>
  </property>
  <widget class="QLabel" name="label_exp">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>17</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expression</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneRes">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>203</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGo">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>318</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stack">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stack</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_res">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>218</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneRev">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_rev">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reverse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneRev_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>264</y>
     <width>491</width>
     <height>40</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_rev_2">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Result&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:7pt;&quot;&gt;(Reverse Temp)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneExp">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>8</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="stackTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>114</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Stack</string>
    </property>
   </row>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
